esp32/src/lib.rs

```diff
@@ -20,7 +20,7 @@
 use core::marker::PhantomData;
 use core::ops::Deref;
 #[doc = r"Number available in the NVIC for configuring priority"]
-pub const NVIC_PRIO_BITS: u8 = 3;
+pub const NVIC_PRIO_BITS: u8 = 0;
```

Comment:
Not sure if we actually use this const anywhere, but this looks odd 😅

Reply:
NVIC is an ARM-only thing, we don't use this constant anywhere and this was just a result of simplifying the SVD generator a bit.